indigo_app/views/works.py

```diff
@@ -561,6 +562,23 @@ def get_works(self, table):
 
         return works
 
+    def get_tasks(self, works, form):
+        tasks = []
+        if form.cleaned_data.get('tasks'):
+            for work in works:
+                task = Task()
+                task.country = self.country
+                task.created_by_user = self.request.user
+                task.save()
```

Comment:
This seems unnecessary... isn't the one lower down before the append enough?

Reply:
we can't assign task.work if task doesn't have an id yet

Reply:
Oh. Django is weird.